### DISCUSSION TITLE
Ventoy and Raspberry pi4/pi5

### DISCUSSION THREAD

**Discussion by msrecko** [ventoy/Ventoy#2751] [General]
Is it technically possible to extend the phenomenal Ventoy tool to run multiboot on pi4/pi5?
Those arm64 platforms do not use UEFI.

Best multiboot for pi only platforms is: https://github.com/procount/pinn
But it would be great to have it under Ventoy ecosystem, because cross platform compatibility.

Tnx.

**Comment by icrygleich** [comment]
couldnt you make a stick with both? .. ill find out soon
